src/utilities/dataLayer.ts

```diff
@@ -4,12 +4,14 @@ import { Serializable } from '../types/utility';
  * @param {String} event            i.e. preorder
  * @param {String} action           i.e. submit
  * @param {Serializable=} context   i.e. { subscribeToNewsletter: true }
+ * @param {Booleam} clear           i.e. false
  */
 export function pushTrackingEvent(
     event: string,
     action: string,
-    context?: Serializable
+    context?: Serializable,
+    clear?: boolean
```

Comment:
je kan hier ook `clear?: boolean = true` van maken dan hoef je hieronder geen if statement te maken.

Reply:
Nee bij TypeScript kan je geen optionele veld en initializer geven.

Reply:
ah oke hmm ja ? zou ook wel weg kunnen maar ik weet niet hoe je typescript type dit dan zou communiceren wanneer je het in project gebruikt. Je kan het ook laten zoals het was.